tools/log-collector/helper.go

```diff
@@ -153,62 +148,37 @@ func filterCSV(csvObjects unstructured.UnstructuredList) unstructured.Unstructur
 // filterCRD returns list of crds created by given set of groups
 func filterCRD(crdObjs unstructured.UnstructuredList) (unstructured.UnstructuredList, error) {
 	crdFilterGroup := []string{TriliovaultGroup, SnapshotStorageGroup, CsiStorageGroup}
-	var filteredCRDObject unstructured.UnstructuredList
+	var filteredCRDObjects unstructured.UnstructuredList
 	for index := range crdObjs.Items {
 		for in := range crdFilterGroup {
```

Comment:
no need of this `for` loop. Just create set of `crdFilterGroup` and check if `crdFilterGroup.Has(crdGroup)`

Reply:
resolved